<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Font Size Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Font Size Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Font Size</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>731</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string>The quick brown fox likes CSE 335</string>
    </property>
   </widget>
   <widget class="EthanLineEdit" name="lineEditSize">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="EthanRadioButton" name="RBarial">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>320</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Arial</string>
    </property>
   </widget>
   <widget class="EthanRadioButton" name="RBtnr">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>320</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Times New Roman</string>
    </property>
   </widget>
   <widget class="EthanRadioButton" name="RBcomicsans">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>320</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Comic Sans</string>
    </property>
   </widget>
   <widget class="EthanSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>360</y>
      <width>381</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="EthanSizeComboBox" name="sizeComboBox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>280</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>small</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>medium</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>big</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Family</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>783</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>EthanSlider</class>
   <extends>QSlider</extends>
   <header location="global">ethanslider.h</header>
  </customwidget>
  <customwidget>
   <class>EthanLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">ethanlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>EthanSizeComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">ethansizecombobox.h</header>
  </customwidget>
  <customwidget>
   <class>EthanRadioButton</class>
   <extends>QRadioButton</extends>
   <header location="global">ethanradiobutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
